gcs/folder.go

```diff
@@ -134,9 +134,7 @@ func (folder *Folder) GetSubFolder(subFolderRelativePath string) storage.Folder
 func (folder *Folder) ReadObject(objectRelativePath string) (io.ReadCloser, error) {
 	path := storage.JoinPath(folder.path, objectRelativePath)
 	object := folder.bucket.Object(path)
-	ctx, cancel := folder.createTimeoutContext()
-	defer cancel()
-	reader, err := object.NewReader(ctx)
+	reader, err := object.NewReader(context.Background())
```

Comment:
What is a variable context? There is no such variable or imported package.

Reply:
I'm not sure that this repository has CI to check the correctness of code.

Reply:
Context is a package. This repository has per-platform tests, they pass on my machine with my GCP credentials...